libraries/ESP8266WebServer/src/detail/RequestHandlersImpl.h

```diff
@@ -15,16 +15,55 @@ class FunctionRequestHandler : public RequestHandler {
     , _uri(uri)
     , _method(method)
     {
+        int numParams = 0, start = 0;
+        do {
+            start = _uri.indexOf("{}", start);
```

Comment:
Why {}? Why not * for glob style notation, or .* for regex notation?

Reply:
The reason that I didn't have chosen for the regex **.*** notation is that it would imply that regex is used. You would expect that ```/users/[0-9]+``` will work too.

The reason that I did use the {} was because I originally wanted a named path arguments:
```
server.on("/users/{userId}/devices/{deviceId}/sensors/{sensorId}", []() {
    String userId = server.pathArg("userId");
    String deviceId = server.pathArg("deviceId");
    String sensorId = server.pathArg("sensorId");
    ...
 });
```
I removed the naming because, you know the index location of the path args.